db/migrate/20200510012452_create_users.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :email, unique: true
```

Comment:
is `unique: true` necessary if the model already enforces uniqueness for the field?

Reply:
It's good to have both DB and model enforcement

Reply:
But crazy, can we make this optional? I think we designed the same model: See "Contact"

https://docs.google.com/document/d/1pbP1NX2bdH_aZGbBJxwxWUfvIf8npKSn0yrDftBGvz8/edit#

Reply:
Done